### ISSUE TITLE
Android  - Search Results Screen - Poor Color Contrast for Search Results Border

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#24]
The color contrast of the border around the individual search results ( ##D1D2D5) is not enough against the white background ( #ffffff ).

The color contrast ratio is 1.51:1 and needs to be 4.5:1 or higher to pass.

Please see the attached screenshot.

![android - search results screen - poor color contrast of search results item border](https://user-images.githubusercontent.com/10423487/57149011-d0613980-6d98-11e9-8720-a0e2f736d415.png)

**Comment by KyleLivingstonDOL** [comment]
Thanks Josh. We will take a look into this.

**Comment by jfc3-dol** [comment]
It's John and you're welcome.

**Comment by KyleLivingstonDOL** [comment]
Thanks John,

So if we increase the contrast -- by making the borders a darker blue, for example -- as long as it meets the 4.5:1, then it is fine?

**Comment by jfc3-dol** [comment]
By making the borders, a darker blue could allow them to pass color contrast depending on the blue.

I use Jonathan Snook's online Colour Contrast Checker ( https://snook.ca/technical/colour_contrast/colour.html ) to make sure the color contrast ratio is 4.5:1, and both of the WCAG 2.0 AA boxes say "YES". All you have to do is put the text and background colors in the foreground and background fields, and it will tell you what you need to know.

**Comment by KyleLivingstonDOL** [comment]
Hi John,

If we update the boxes in black, would that suffice to meet the color contrast requirements?

Thanks,
Kyle

**Comment by jfc3-dol** [comment]
Yes, you could make them black on the white background to pass, or you could use a dark charcoal gray ( #3d3d3d ) on a white ( #ffffff ) background which gives you enough color contrast but is not as jarring on the eyes as using black.

https://snook.ca/technical/colour_contrast/colour.html#fg=3D3D3D,bg=FFFFFF

**Comment by KyleLivingstonDOL** [comment]
> Yes, you could make them black on the white background to pass, or you could use a dark charcoal gray ( #3d3d3d ) on a white ( #ffffff ) background which gives you enough color contrast but is not as jarring on the eyes as using black.
> 
> https://snook.ca/technical/colour_contrast/colour.html#fg=3D3D3D,bg=FFFFFF

Thanks John. We will do that. We will make the borders a dark charcoal gray.

**Comment by jfc3-dol** [comment]
The border for each search result now has enough color contrast, so I'm going to close this issue.
